index.rst

```diff
@@ -0,0 +1,190 @@
+
+Psf libraries were created for “raw seeing” values of 0.5, 0.7, and 0.9 arcseconds and for LSST filters (f2 and f3).  Most of the tests during this cycle were run using the f2_0.7 library.
+
+Using Great3Sims:
+-------------------------
+
+We used the Great3sims package to produce yaml files which in turn drive the GalSim creation of galaxy images.  Great3Sims produces a catalog of galaxies to be constructed using parameterized models and randomly drawn parameters.  A constant shear is then applied for each “subfield”, with the shear and shear angle drawn at random from a specified range.  In this respect, our galaxy sample was the same as that used for the “control” branch of the great3sims test.
+
+One important modification to Great3Sims was to allow the point spread function for each galaxy to be sampled for our Psf Library rather than from a parameterized Psf model.  The PhoSim Psf libraries described above were used to select a Psf image  GalSim during the creation of the galaxy images.  So instead of convolving the test galaxies with a parameterized Psf model, we supplied the Psf for each model as a fits image (typically 67x67).  The fits image was supplied to GalSim using the InterpolatedImage input type.
+
+The great3sims “control/ground/constant” tests provided for 200 subfields, each with a constant shear and shear angle.  There are 10000 galaxies per subfield.  Each subfield is assigned its own shear and shear angle at random.
+
+In some of our early tests, we made the subfields smaller (1024 galaxies), but made the number of subfields larger (1024 subfields for each shear value).  This was done in an attempt to sample the error in the absolute value of the shear.  Some of these early tests were also larger (6 million galaxies), which made our tests more sensitive, but possibly not realistic.
+
+Using GalSim:
+-------------------------
+
+GalSim is driven by the Great3sims program, which creates a yaml file for GalSim. The profile and shear information if supplied for each galaxy through epoch catalog.  Except for the introduction of a Psf image, we did not alter this relationship between Great3sims and GalSim.
+
+The epoch catalog is typically named epoch_catalog-nnn-m.fits, with nnn the ordinal number of the subfield, and m the epoch (always 0 in our tests).  GalSim creates a matching image-nnn-m.fits with a 96x96 pixel image for each galaxy in the sample.
+
+Doing Measurements:
+-------------------------
+
+The goal of the framework is to compare the results of a shape measurement algorithm with the known constant shear values which are produced with the Great3sims/GalSim simulation.  The algorithm must be housed in an lsst.meas.base measurement plugin so that it can be used by the LSST DM Stack.
+
+Our intial tests were with the CModel algorithm in lsst.meas.modelfit package, but this framework could be used to test any shape algorithm which is housed in a measurement plugin.
```

Comment:
I think this point is key, and it should be made explicit in the introduction -- see my earlier comment.